packages/compass-global-writes/src/plugin-title.tsx

```diff
@@ -49,7 +55,7 @@ export const PluginTitle = ({ showWarning }: { showWarning: boolean }) => {
               }}
             >
               <Icon
-                glyph="ImportantWithCircle"
+                glyph={showWarning ? 'Warning' : 'ImportantWithCircle'}
                 aria-label="warning"
```

Comment:
Could we make this message and the color of the icon variable as well?

Reply:
Is this what you mean?

Error: 
![image](https://github.com/user-attachments/assets/8fe733a1-b12b-40e2-a0bf-8812464cfa41)

Warning:
![image](https://github.com/user-attachments/assets/c9805fb7-8fe5-4a5d-94bd-aed63643b441)

Reply:
Code quality question, it seems weird that the properties of the Icon are so coupled to each other, should we just pull out a WarningIcon and ErrorIcon or something pre-parameterized? Does this pattern come up a lot in compass?

Reply:
You're right, with this many differences it makes sense to move the icons into their own function components so that the main render is more readable. And yes that's a common pattern, not only in compass. If your question is more towards reusable components in Compass, check the `compass-components` package. But I wouldn't put these in there as the warning = ImportantWithcircle and error = Warning is a bit odd. I'd just keep them in the same file.